tech/languages/python/multiple-pythons.md

```diff
@@ -187,6 +188,58 @@ Type "help", "copyright", "credits" or "license" for more information.
 To learn more about virtualenvs, visit
 [The Hitchhiker's Guide to Python](http://docs.python-guide.org/en/latest/dev/virtualenvs/).
 
+### Python 2.6, Jython
+
+The versions of virtualenv and tox packaged in Fedora do not support Python 2.6
+and Jython, because they are extremely old. However, there is a trick on how to
+use older virtualenv/tox if you really need to support such old interpreters.
```

Comment:
Where do I get the old one? Is it really something you'd recommend? If not, please ommit.